protocols/luci-proto-wireguard/htdocs/luci-static/resources/protocol/wireguard.js

```diff
@@ -709,6 +709,7 @@ return network.registerProtocol('wireguard', {
 				'[Interface]',
 				'PrivateKey = ' + prv,
 				eport ? 'ListenPort = ' + eport : '# ListenPort not defined',
+				dns && dns.length ? 'DNS = ' + dns.join(', ') : '# DNS not defined',
```

Comment:
According to my tests, wireguard clients do not like unfinished lines that end with a comma `,`, so one must test for the array size and only if it is `>= 1` should one `join`.

Reply:
Or as the man page details, add one `DNS = ` line per dns entry. 

The man page says `May be specified multiple times.`

Reply:
@nevumx if you can address this, we can get this merged. 

If necessary, use nested ternaries to test for `.length > 1`